pages/jobs/index.js

```diff
@@ -1,65 +1,41 @@
-import JobsContext from "../../context/JobsContext"
-import JobPostItem from "../../components/navigation/JobPostItem"
-import Footer from "../../components/Footer"
-import Head from "../../components/Head"
-import Filter from "../../components/navigation/Filter"
-import AnnouncementHero from "../../components/campaigns/AnnoucementHero"
+import { useContext, useState } from "react";
+import NavBar from "../../components/navigation/NavBar";
+import Container from "../../components/layout/Container";
+import JobsContext from "../../context/JobsContext";
+import JobPostItem from "../../components/navigation/JobPostItem";
+import Footer from "../../components/Footer";
+import Head from "../../components/Head";
+import Filter from "../../components/navigation/Filter";
+import AnnouncementHero from "../../components/campaigns/AnnoucementHero";
 import Empty from "../../components/illustrations/empty";
 
 export default function JobsIndexPage() {
-  const { navItems } = useContext(JobsContext)
+  const { navItems } = useContext(JobsContext);
   navItems.map((job) => {
-    if(new Date() > new Date(job.closingOn)){
-      navItems.splice(job, 1)
+    if (new Date() > new Date(job.closingOn)) {
+      navItems.splice(job, 1);
     }
-  })
-  const [posts, setPosts] = useState(navItems.sort((i1, i2) => {
-    const i1Date = new Date(i1.date)
-    const i2Date = new Date(i2.date)
-    return i2Date - i1Date
-  }));
+  });
+  const [filter, setFilter] = useState(null);
+  const [posts, setPosts] = useState(
+    navItems.sort((i1, i2) => {
+      const i1Date = new Date(i1.date);
+      const i2Date = new Date(i2.date);
+      return i2Date - i1Date;
+    })
+  );
 
   const onFilter = (data) => {
     setPosts(data);
-  }
-const body=`---
-title: 'Job Title'
-date: MM/DD/YYYY (current date)
-category: job category
-closingOn: MM/DD/YYYY
-contact: provide link to oryginal job posting or a contact email address
-company: 
-  name: 'YourCompanyName'
-  logoUrl: /img/logos/companies/YourCompanyName.png
-  # Add your image in /pages/img/logos/companies
----
-## About the team
-Let potential employees knows what it'll feel like to joining your team in this section
-
-## TL;DR
-Enter list of perks for potential employees in this section.
-E.g.
-* :muscle: Growing team
-* :house_with_garden: Fully remote job
-* :money_mouth_face: Great salary and compensation package
-* :mountain_snow: Unlimited paid time off
-
-## About the job
-Let the potential employee knows about the job in this section
-
-## About you
-The potential employees would love to know more about you. Tell them more about you in this section
-
-## Pay and benefits
-Let the potential employees knows what comes with joining your team in this section.
-
-Join us!
-`;
-const jobPostUrl = encodeURIComponent(body);
-const hasPosts = Object.keys(posts).length;
+  };
+  const toFilter = [
+    {
+      name: "category",
+    },
+  ];
+  const jobPostUrl =
```

Comment:
Why you change the `body` to encoded URI?

Reply:
@magicmatatjahu I didn't want the file to look overwhelming. Or did you guys do it so that it can be modified?

Reply:
Yah, it was done because there should be possibility to modify it :)